.github/workflows/ci.yml

```diff
@@ -0,0 +1,50 @@
+name: CI
+
+on:
+  - pull_request
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    env:
+      COMPOSER_HOME: /home/runner/.composer/cache
```

Comment:
This might not be necessary anymore

```suggestion
```

Reply:
That's right. It defaults to `/home/runner/.composer/cache` anyway